module/VuFind/src/VuFind/Config/ConfigLoader.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * Configuration loader
+ * Note: This is only used to load the caching configuration in the config.ini without file caching and should
+ * not be used otherwise. Use the ConfigManager instead.
```

Comment:
Would something like this be a little clearer / more general?

```suggestion
 * Note: This class supports the ConfigManager but should not be called directly except in rare situations
 * where the ConfigManager must be bypassed to avoid circular dependencies (as when setting up the
 * Cache Manager). Unless you have a very good reason, always use CacheManager instead.
```